src/CabanaPD_Solver.hpp

```diff
@@ -294,7 +274,12 @@ class SolverElastic
             log( std::cout, step, "/", num_steps, " ", std::scientific,
                  std::setprecision( 2 ), step * dt );
 
-            total_time = total_timer.seconds();
+            double total_time = _total_timer.time();
+            double comm_time = comm->time();
+            double integrate_time = integrator->time();
+            double force_time = force->time();
+            double energy_time = force->timeEnergy();
+            double output_time = particles->time();
```

Comment:
@streeve Why "output_time" is "particles->time"?

Reply:
Because the particles class does the output